packages/grid/src/ExpandableGridModel.ts

```diff
@@ -30,8 +33,23 @@ export interface ExpandableGridModel extends GridModel {
    * Change the expanded status of an expandable row
    * @param row Row to expand
    * @param isExpanded True to expand the row, false to collapse
+   * @param expandDescendants True to expand nested rows, false otherwise
    */
-  setRowExpanded(row: ModelIndex, isExpanded: boolean): void;
+  setRowExpanded(
+    row: ModelIndex,
+    isExpanded: boolean,
+    expandDescendants?: boolean
+  ): void;
+
+  /**
+   * Expand all rows
+   */
+  setExpandAll(): void;
+
+  /**
+   * Collapse all rows
+   */
+  setCollapseAll(): void;
```

Comment:
Also these methods should just be called `expandAll` and `collapseAll`, no need to call them `setExpandAll` or `setCollapseAll`.